### PR TITLE
fixed the latest asked changes

### PR DESCRIPTION
**Pull request by Mohsinsiddi**
merging the latest changes with dotmatrix main branch

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -1,65 +1,69 @@
...
-                </div>
+  return (
+    <div className={styles.footer}>
+      <div className={styles.upperfooter}>
+        <div className={styles.upperleft}>
+          <div className={styles.nervos}>
+            <img src={nervos} alt="Nervos"></img>
+          </div>
+          <div className={styles.goddwoken}>
+            <img src={Goddwoken} alt="Goddwoken"></img>
```
Comment (nblogist) [inline src/components/Footer/Footer.js]: Misspell of godwoken

**Review by nblogist — CHANGES_REQUESTED** [review]

**Review by Mohsinsiddi — COMMENTED** [review]
seems goo d to me now

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (Mohsinsiddi) [reply src/components/Footer/Footer.js]: made changes here

Comment (Mohsinsiddi) [reply src/components/Footer/Footer.js]: @nblogist please review

**Review by nblogist — APPROVED** [review]

**Pull request merged**